app/src/main/java/com/example/game/models/Attendance.java

```diff
@@ -0,0 +1,58 @@
+package com.example.game.models;
+
+import com.parse.Parse;
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+import com.parse.ParseUser;
+
+import org.parceler.Parcel;
+
+/*
+Parse Model: Attendance (User - event)
+| Property | Type     | Description |
+| -------- | -------- | -------- |
+|objectId     | String    | unique id for the user post |
+|user |	Pointer to User | the person who created the post|
+|event |	Pointer to Event| Comments to a post|
+|attendStatus | Boolean |whether attending or not attending|
+|likeStatus | Boolean | whether likes or not|
+|createdAt|	DateTime|	date when the first interaction is created (default field)|
+|updatedAt|	DateTime|	date when comment is last updated (default) field)|
+ */
+@Parcel(analyze = {Attendance.class})
+@ParseClassName("Attendance")
+public class Attendance extends ParseObject {
+    public static final String KEY_USER = "user";
+    public static final String KEY_EVENT = "event";
+    public static final String KEY_ATTEND_STATUS = "attendStatus";
+    public static final String KEY_LIKE_STATUS = "likeStatus";
+    public static final String KEY_CREATED_AT = "createdAt";
```

Comment:
Seems you only access those from this class so mark as private

Reply:
I will access them later from one of my fragments.

Reply:
I will access them later in one of my fragments. I will need to access them without constructing an object.